### PR TITLE
8.0 product_unique_serial and stock_no_negative modules

### PR DESCRIPTION
**Pull request by osvalr**
This two modules were added by @moylop260 to control product serial lot number with a unique relation, adding a restriction to avoid stock moves with quantity different than 1, as the module describes and discussed in this old PR https://github.com/OCA/stock-logistics-workflow/pull/64. This is an up-to-date modules, adapted to api 8.0 waiting for merge in the pull request pointed before 
@pedrobaeza @gurneyalex @nhomar

### CODE REVIEW

**Comment by osvalr** [comment]
Hi @moylop260 & @nhomar, 

I propose this PR to close the [old one](https://github.com/OCA/stock-logistics-workflow/pull/64) and use this instead. Thanks to both of your guys

**Comment by yaniaular** [comment]
IMHO LGTM :+1: , just change the methods to the API 8.0 and make rebase. Thanks @osvalr

File: .travis.yml [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
+      - python-simplejson
+      - python-yaml
 
 python:
   - "2.7"
 
 env:
   global:
-    - VERSION="8.0" TESTS="0" LINT_CHECK="0"
+  - LINT_CHECK=0 TESTS="0" VERSION="8.0" ODOO_REPO="vauxoo/odoo"
```
Comment (eLBati) [inline .travis.yml:22]: @osvalr why `vauxoo/odoo`?

Comment (osvalr) [inline .travis.yml:22]: Hi @eLBati 
I've missed that, I'll fix that right away!

Comment (eLBati) [inline .travis.yml:22]: Ok, please wait: I'm preparing a PR with other things also

Comment (osvalr) [inline .travis.yml:22]: Thanks for noticing @eLBati

**Pull request merged**